### PR TITLE
save a solution even if infeasible

### PR DESCRIPTION
**Pull request by chriscoey**
to avoid an UndefRefError I'm getting in Pajarito tests

```
  Got an exception of type UndefRefError outside of a @test
  UndefRefError: access to undefined reference
   in getsolution(::ConicNonlinearBridge.NonlinearToConicBridge) at /home/chriscoey/.julia/v0.5/ConicNonlinearBridge/src/nonlinear_to_conic.jl:254
   in solve_cbf(::String, ::String, ::Pajarito.PajaritoSolver, ::Bool) at /home/chriscoey/.julia/v0.5/Pajarito/test/conictest.jl:49

```

### CODE REVIEW

**Comment by mlubin** [comment]
Meh, the proper fix is https://github.com/JuliaOpt/MathProgBase.jl/issues/156, but until then I'll merge this if it doesn't break anything. The main issue is that we don't currently have a way to query a model and ask if a solution is available.

**Comment by chriscoey** [comment]
sure but other solvers don't usually give you an UndefRefError when you ask for a solution

**Comment by mlubin** [comment]
Pretty sure cplex will throw an error, for example, if it doesn't have a solution to give you.

File: src/nonlinear_to_conic.jl [hunk 1]
```diff
@@ -255,7 +250,4 @@ MathProgBase.getsolution(m::NonlinearToConicBridge) = m.solution[1:size(m.A_ini,
 MathProgBase.getsolvetime(m::NonlinearToConicBridge) = MathProgBase.getsolvetime(m.nlp_model)
 
 MathProgBase.numvar(m::NonlinearToConicBridge) = m.numVar
-function MathProgBase.numconstr(m::NonlinearToConicBridge)
-    # is numconstr well defined for conic models?
```
Comment (mlubin) [inline src/nonlinear_to_conic.jl:259]: Do we need to remove this comment? It's there for a reason

Comment (chriscoey) [reply src/nonlinear_to_conic.jl:259]: OK I thought it was well enough defined because there is a way to query number of constraints in conic models

Comment (mlubin) [reply src/nonlinear_to_conic.jl:259]: What's the way?

**Pull request merged**